### DISCUSSION TITLE
Ability to sync VOD/replay feeds

### DISCUSSION THREAD

**Discussion by gc04caps** [f1multiviewer/issue-tracker#14] [Feature requests] [answered]
Feature request: ability to sync/re-sync multiple feeds for video-on-demand/replay. Ideally this would keep in sync with FF/Rewind/Seek operations or be a button or hotkey to resync

**Comment by Petarkco** [comment]
Two ideas could be used:

One would be to borrow from the https://github.com/najdek/f1tv_plus project and use user created/saved offsets for each stream (and have them available to all users).

The second was to use the English/International commentary that appears on all streams as a way to sync them but I understand that it may not always work and and custom offsets may be a backup in those cases.

**Reply by loheiman** [reply]
Please! As someone who lives in the US, I watch all of the races via replay. Not being able to sync all the video streams after pausing prevents me from being able to use pause at all.

**Reply by jtokash** [reply]
@loheiman  This is now supported in replay mode by pressing the s key.  Doesn't work in live yet.

**Reply by Aliensarecoming** [reply]
If this doesn't work - delete your C:\Users\%username%\AppData\Roaming\MultiViewer for F1\ folder.

**Comment by f1multiviewer** [comment]
For those here looking for a solution that "works" but isn't perfect:

Press `S` (wait approx. 10s after starting the session). Other players will sync to the same playback position. Improvements are coming to make this *perfect*!

**Comment by JJWatMyself** [comment]
Best way to sync manually is now documented by the head chef.

https://discord.com/channels/982375740764983366/983777853055312002/989420360707473419
[](https://discord.com/channels/982375740764983366/983777853055312002/989420360707473419)

**Comment by f1multiviewer** [comment, marked as answer]
For users: if the race has been pre-synced by someone, you can just open the streams you want to watch and press `S` to sync them (best to pause them all by pressing `Shift+Space`, then `S`, then `Shift+Space` to play them all.

I'll add the explanation from Github to the docs, I hope that adding syncs will soon be something that's super easy, but for now we already have a good amount of races synced.

**Comment by Petarkco** [comment]
To add to this maybe make a setting were if a session has pre-set offsets automatically adjust the streams so they stay in sync unless it's been manually changed. 

To have on having to pause everything and getting it to resync each time something goes out of sync.

**Reply by f1multiviewer** [reply]
Do they go out of sync for you? I tried it myself and I didn't have that issue. But yeah, maybe at some point, but a separate feature-request would be advisable so I don't loose track :)
